diffpy/snmf/tests/test_subroutines.py

```diff
@@ -14,3 +15,19 @@ def test_objective_function(to):
     actual = objective_function(to[0][0], to[0][1], to[0][2], to[0][3], to[0][4], to[0][5])
     expected = to[1]
     assert actual == pytest.approx(expected)
+
+
+tgso = [
+    ([.25, [6.55, .357, 8.49, 9.33, 6.78, 7.57, 7.43, 3.92, 6.55, 1.71], 10], [6.55, 6.78, 6.55, 0, 0, 0, 0, 0, 0, 0]),
+    ([1.25, [-11.47, -10.688, -8.095, -29.44, 14.38], 5], [-11.47, -10.8444, -9.1322, -16.633, -20.6760]),
+    ([.88, [-11.47, -10.688, -8.095, -29.44, 14.38], 5], [-11.47, -10.3344, -13.9164, -11.5136, 6.5364]),
+    ([.88, [1, 2, 3, 4, 5, 6, 7, 8, 9, 10], 10], [1, 2.1364, 3.2727, 4.4091, 5.5455, 6.6818, 7.8182, 8.9545, 9.0909, 0])
+
+]
+
```

Comment:
I prefer to remove these blank lines in all the tests so it is easy to see which sets of inputs correspond to which tests.

Reply:
I will remove ths blank lines